roles/opennms_backup/tasks/ubuntu_debian.yml

```diff
@@ -0,0 +1,78 @@
+---
+- name: Back up OpenNMS Horizon opt folder
+  community.general.archive:
+    path:
+    - /usr/share/opennms/*
+    dest: "{{ onms_backup_folder }}/{{ timestamp }}_opennms-opt.tar.gz"
+    exclude_path:
+    - /usr/share/opennms/data/cache/*
+    format: gz
+
+- name: Back up OpenNMS Horizon etc folder
+  community.general.archive:
+    path:
+    - /etc/opennms/*
+    dest: "{{ onms_backup_folder }}/{{ timestamp }}_opennms-etc.tar.gz"
+    format: gz
+
+- name: Back up OpenNMS Horizon lib folder
+  community.general.archive:
+    path:
+    - /var/lib/opennms/*
+    dest: "{{ onms_backup_folder }}/{{ timestamp }}_opennms-var-lib.tar.gz"
+    format: gz
+
+- name: Back up OpenNMS Horizon usr/share/java folder
+  community.general.archive:
+    path:
+    - /usr/share/java/opennms/*
+    dest: "{{ onms_backup_folder }}/{{ timestamp }}_opennms-usr-share-java.tar.gz"
+    format: gz
+
+- name: Create latest opt folder tar
+  ansible.builtin.copy:
+    remote_src: true
+    src: "{{ onms_backup_folder }}/{{ timestamp }}_opennms-opt.tar.gz"
+    dest: "{{ onms_backup_folder }}/latest_opennms-opt.tar.gz"
+
+- name: Create latest etc folder tar
+  ansible.builtin.copy:
+    remote_src: true
+    src: "{{ onms_backup_folder }}/{{ timestamp }}_opennms-etc.tar.gz"
+    dest: "{{ onms_backup_folder }}/latest_opennms-etc.tar.gz"
+
+- name: Create latest var-lib folder tar
+  ansible.builtin.copy:
+    remote_src: true
+    src: "{{ onms_backup_folder }}/{{ timestamp }}_opennms-var-lib.tar.gz"
+    dest: "{{ onms_backup_folder }}/latest_opennms-var-lib.tar.gz"
+
+- name: Create latest usr-share-java folder tar
+  ansible.builtin.copy:
+    remote_src: true
+    src: "{{ onms_backup_folder }}/{{ timestamp }}_opennms-usr-share-java.tar.gz"
+    dest: "{{ onms_backup_folder }}/latest_opennms-usr-share-java.tar.gz"
+
+- name: Grep OpenNMS user ID
+  shell:
+    cmd: "grep opennms /etc/passwd > {{ onms_backup_folder }}/{{ timestamp }}_opennms-passwd.txt"
+  ignore_errors: yes
+  changed_when: false
+
+- name: Grep OpenNMS group ID
+  shell:
+    cmd: "grep opennms /etc/group > {{ onms_backup_folder }}/{{ timestamp }}_opennms-group.txt"
+  ignore_errors: yes
+  changed_when: false
+
+- name: Create latest passwd file
```

Comment:
Would be cool to have an handler that always does this `latest` copy process. But I didn't find a solution for this right now. Also, we have those tasks in CentOS and Ubuntu tasks.